projects/self-creation-portal/src/app/app.component.html

```diff
@@ -5,3 +5,5 @@
 <lib-card [list]="list"></lib-card>
 
 <lib-search (searchEvent)="receiveSearchResults($event)" [maxLength]="150"></lib-search>
+
+<lib-pagination totalCount="6" (paginatorChanged)="onPaginatorChange($event)"></lib-pagination>
```

Comment:
did you checked the complete pagination flow with json data to check pagination and events ?

Reply:
yes i have checked its working